Source/ThirdParty/ANGLE/src/libANGLE/renderer/vulkan/ContextVk.cpp

```diff
@@ -5654,6 +5654,14 @@ angle::Result ContextVk::syncState(const gl::Context *context,
                         case gl::State::EXTENDED_DIRTY_BIT_SHADING_RATE:
                             mGraphicsDirtyBits.set(DIRTY_BIT_DYNAMIC_FRAGMENT_SHADING_RATE);
                             break;
+                        case gl::State::EXTENDED_DIRTY_BIT_VRR:
+                            // Nothing to do until ANGLE_variable_rasterization_rate_metal is
+                            // implemented.
+                            break;
+                        case gl::State::EXTENDED_DIRTY_BIT_VRR:
```

Comment:
This case statement is duplicated. Surprised this compiled.

Reply:
That wasn't intentional. Rebase issue?